pkg/cli/stress_test.go

```diff
@@ -129,6 +125,9 @@ func TestSressTooManyOpenFiles(t *testing.T) {
 		}
 		wg.Wait()
 
+		_, err = setMaxOpenFilesLimit(initialLimit)
```

Comment:
This is for my own curiosity - is it needed to reset the resource limit, since the previously set value will be available only for the current process ?